ipfscluster_test.go

```diff
@@ -205,26 +205,37 @@ func createClusters(t *testing.T) ([]*Cluster, []*test.IpfsMock) {
 	for i := 1; i < nClusters; i++ {
 		cfgs[i].Bootstrap = []ma.Multiaddr{bootstrapAddr}
 	}
+	time.Sleep(200 * time.Millisecond)
 
 	// Start the rest
-	var wg sync.WaitGroup
+	// Doing this in parallel causes libp2p dial backoffs
```

Comment:
it looks like this changed from parallel to sequential which also makes sense in terms of avoiding collisions and backoffs.  Is the comment off or am I misinterpreting the code?

Reply:
oh, the comment just explains why it's not in parallel. ill change it